curriculum/js1/8.js

```diff
@@ -5,8 +5,11 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,b,gcd=1,div=1) => {
+  if (div>a || div>b) return gcd;
+   if (a%div===0 && b%div===0) gcd=div;
+return solution(a,b,gcd,div+1);
+
```

Comment:
Minor, general comment on horizontal spacing: most code styles will recommend a horizontal space after commas and around equal signs to make the code easier to read.

E.g.:
```javascript
solution(a, b, gcd, div+1)
```
```javascript
if (a%div === 0 && b%div === 0) gcd = div;
```

Reply:
sure ill fix it.